usecases/schema/property.go

```diff
@@ -54,9 +54,9 @@ func (h *Handler) AddClassProperty(ctx context.Context, principal *models.Princi
 	}
 
 	existingNames := make(map[string]bool, len(class.Properties))
-	if !merge {
-		for _, p := range class.Properties {
-			existingNames[strings.ToLower(p.Name)] = true
+	for _, prop := range class.Properties {
+		if !merge {
```

Comment:
Since `merge` doesn't change, would it make sense to move it outside the loop.

Reply:
sure, it makes more sense when it is outside. I've tried with other implementation, messed up with reverting all the changes.

Reply:
changed